rwengine/src/ai/CharacterController.cpp

```diff
@@ -292,8 +266,44 @@ bool Activities::EnterVehicle::update(CharacterObject *character,
     return false;
 }
 
+bool Activities::InVehicle::update(CharacterObject* character,
+                                   CharacterController* controller, float dt) {
+
+    auto &d = character->getMovement();
+
+    if (character->getCurrentSeat() == 0) {
+        character->getCurrentVehicle()->setSteeringAngle(d.y);
+
+        if (std::abs(d.x) > 0.01f) {
+            character->getCurrentVehicle()->setHandbraking(false);
+        }
+        character->getCurrentVehicle()->setThrottle(d.x);
+    }
+
+
+    // If character is idle in vehicle, try to close the door.
+    auto v = character->getCurrentVehicle();
```

Comment:
dunno what these return but shouldn't they be auto&'s to prevent creating copies?

Reply:
It returns a pointer